compiler/parser/chapel.ypp

```diff
@@ -266,6 +269,9 @@
 
 %debug
 %verbose
+// Thes would print out expected token names
```

Comment:
"Thes" -> "These" ?
Otherwise Paul will come after you with the spell checker. :)